server/routerlicious/kubernetes/routerlicious/templates/fluid-configmap.yaml

```diff
@@ -120,7 +120,15 @@ data:
             "socketIo" : {
                 "perMessageDeflate": {{ .Values.alfred.socketIo.perMessageDeflate}}
             },
-            "getDeltasRequestMaxOpsRange": {{ .Values.alfred.getDeltasRequestMaxOpsRange }}
+            "getDeltasRequestMaxOpsRange": {{ .Values.alfred.getDeltasRequestMaxOpsRange }},
+            "getDeltasEndingGapDisableFallback": {
+                "enableTenants": {{ toJson .Values.alfred.getDeltasEndingGapDisableFallback.enableTenants }}
```

Comment:
Are these a list of PP list? If so there is one already defined in shared config in FRS, maybe we can use that one?

Reply:
no, this will be the list of tenants used for stress tests by the ff team